packages/adapter-nuxt/src/hooks/slice-create.ts

```diff
@@ -74,42 +76,45 @@ const createComponentFile = async ({ dir, data, helpers, options }: Args) => {
 		`;
 	}
 
-	if (options.format) {
-		contents = await helpers.format(contents, filePath, {
-			prettier: { parser: "vue" },
```

Comment:
So this is not required here 🤔

Assuming yes since tests are still passing(?)
https://github.com/prismicio/slice-machine/blob/aa/adapter-nuxt-fs-helpers/packages/adapter-nuxt/test/plugin-slice-create.test.ts#L194-L238

Reply:
Correct; `writeSliceFile()` (and all other `write*File()` helpers) applies formatting if the `format` option is set to `true`. Prettier automatically sets the `vue` parser if the file's extension ends in `.vue`.